src/tests/eolian/data/class_simple_ref.c

```diff
@@ -17,7 +17,7 @@ __eolian_class_simple_a_set_reflect(Eo *obj, Eina_Value val)
    return r;
 }
 
-EAPI EAPI_WEAK EFL_FUNC_BODYV(efl_canvas_object_simple_a_set, Eina_Bool, EINA_TRUE /* true */, EFL_FUNC_CALL(value), int value);
+ EFL_FUNC_BODYV(efl_canvas_object_simple_a_set, Eina_Bool, EINA_TRUE /* true */, EFL_FUNC_CALL(value), int value);
```

Comment:
There is a blank space left in every `EAPI EAPI_WEAK` replacement in the last commit.

Reply:
oh boy....

Reply:
done, thanks